### PR TITLE
Remove focus after cancelling modal

### LINKED ISSUE #446: After adding a server on the Settings page, focus remains at "Add new server" link
I confirm (by marking "x" in the [ ] below): 

- [X] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [X] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [X] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md). 

---

**Summary**
After adding a server on the Settings page, focus remains at "Add new server" link

**Steps to reproduce**

1) Go to File > Settings
2) Click "Add new server"
3) Hit Cancel

**Expected behavior**
Focus should be away from "Add new server" link

**Observed behavior**
Focus is on the "Add new server" link
image

PS: This reproduces for "Edit" and "Remove" buttons too

### PR DESCRIPTION
**Pull request by wvds**
Closes #446 

Before submitting, please confirm you've
 - [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
 - [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
 - [x] executed `npm run lint:js` for proper code formatting

Please provide the following information:

**Summary**
After closing the modal the focus of the clicked link will be removed

**Issue link**
#446

### CODE REVIEW

File: src/browser/components/TeamListItem.jsx [hunk 1]
```diff
@@ -9,9 +9,11 @@ class TeamListItem extends React.Component {
   }
 
   handleTeamRemove() {
+    document.activeElement.blur();
     this.props.onTeamRemove();
   }
   handleTeamEditing() {
+    document.activeElement.blur();
```
Comment (yuya-oc) [inline src/browser/components/TeamListItem.jsx]: In the team list, `TeamList.jsx` controls modals. So I think it should call document.activeElement.blur().

File: src/browser/components/TeamListItem.jsx [hunk 2]
```diff
@@ -9,9 +9,11 @@ class TeamListItem extends React.Component {
   }
 
   handleTeamRemove() {
+    document.activeElement.blur();
```
Comment (yuya-oc) [inline src/browser/components/TeamListItem.jsx]: In the team list, `TeamList.jsx` controls modals. So I think it should call document.activeElement.blur().

**Comment by yuya-oc** [comment]
Thanks @wvds, the behavior looks good! For better code, would you review my comments?

**Comment by yuya-oc** [comment]
https://circleci.com/gh/mattermost/desktop/1049#artifacts

**Comment by wvds** [comment]
@jasonblais @yuya-oc I've made the changes to the pull request :-)
CI is also fixed: https://circleci.com/gh/mattermost/desktop/1052

**Comment by jasonblais** [comment]
Thanks @wvds!! :)

**Pull request merged**